crates/apps/src/lib/client/rpc.rs

```diff
@@ -1058,8 +1058,6 @@ pub async fn query_bonded_stake<N: Namada>(
                 get_validator_stake(context.client(), epoch, &validator).await;
             match stake {
                 Some(stake) => {
-                    // TODO: show if it's in consensus set, below capacity, or
-                    // below threshold set
```

Comment:
we do have a separate cmd for this, but it could be nice (and ez) to add this below the `display_line!`:

```rust
                    query_and_print_validator_state(
                        context,
                        args::QueryValidatorState {
                            query: args.query,
                            validator,
                            epoch: args.epoch,
                        },
                    )
                    .await;
```